src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -136,18 +145,16 @@ const UploadDocumentForm = () => {
               onChange={(newDocDescription) => setDocDescription(newDocDescription.target.value)}
               placeholder="Add a description here"
             />
-          </FormControl>
-          <br />
-          <br />
-          <FormControl fullWidth>
+            <br />
             <Button
-              variant="contained"
+              variant="outlined"
```

Comment:
Very minor thing, but I we can keep this `contained` when `file` is missing, then if `file` is chosen, we can switch this back to `outline`.

It'll help draws the eye to the `Choose file` button at the beginning then to `Upload file` after a file is chosen.

Reply:
I tested this and REALLY like it...  basically shows the user that all green buttons are good and should be done. And almost creates a step by step guide.  1st step is choose file, 2nd step is upload.

https://github.com/codeforpdx/PASS/assets/71395931/5b08988f-adbc-4e34-8a45-583cf29a307e

I did this using a ternary on the "variant" of the `choose file` button, checking for the `file` state.